dot-gen/src/main/java/com/wipro/ats/bdre/ldg/LineageDotGen.java

```diff
@@ -30,16 +31,22 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LineageDotGen.class);
     private static String defaultHiveDbName = LineageConstants.defaultHiveDbName;
-
-    private static final String[][] PARAMS_STRUCTURE = {
-            {"p", "sub-process-id", "Sub Process id of the step"},
-    };
+    private static final String PARENTPROCESSID = "parent-process-id";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "PARENTPROCESSID" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)